backend/test/modules/doubles-tournament/core/application/DoublesTournamentsQuerySide.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { testDoublesTournamentsModule } from './TestDoublesTournamentsModule';
+import { FromListIdGeneratorStub } from '../../../../test-support/shared/core/FromListIdGeneratorStub';
+import { FindAllDoublesTournaments } from '../../../../../src/modules/doubles-tournament/core/application/query/FindAllDoublesTournaments';
+import { CreateTournamentWithTeams } from '../../../../../src/modules/doubles-tournament/core/application/command/CreateTournamentWithTeams';
+import { DoublesTournament } from '../../../../../src/modules/doubles-tournament/core/domain/DoublesTournament';
+import { TournamentTeam } from '../../../../../src/modules/doubles-tournament/core/domain/TournamentTeam';
+import { FindDoublesTournamentById } from '../../../../../src/modules/doubles-tournament/core/application/query/FindDoublesTournamentById';
+
+describe('Create Doubles Tournament | Query Side', () => {
```

Comment:
Albo zmień descirbe albo nazwę pliku :) raczej jestem za nazwa testu, który będzie miał nazwę modułu bo teraz to "Creates" nie wiadomo do czego jest.